mocks/digdir-mock/src/main/java/no/nav/digdir/DigdirKrrProxyMock.java

```diff
@@ -45,7 +50,27 @@ public Response hentKontaktinformasjon(@HeaderParam(HEADER_NAV_PERSONIDENT) @Not
         return Response.ok(kontaktinformasjon).build();
     }
 
-    public String hentUtForetrukketSpråkFraBruker(String fnr) {
+    @POST
+    @Path("/rest/v1/personer")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Operation(description = "Henter kontaktinformasjon for person")
+    public Response hentKontaktinformasjon(@Valid List<@NotNull String> personidenter,
```

Comment:
1. Kommer det ikke et objekt som inneholder en liste ? 
2. Fjerne .header(NavHeaders.HEADER_NAV_PERSONIDENT, fnr) - det er motivasjonen for omleggingen

Reply:
1. Stemmer. Blingset litt og har fikset det nå.
2. Det skal jeg fjerne.